src/classes/QueryPredicate.cls

```diff
@@ -0,0 +1,60 @@
+/*************************************************************************************************
+* This file is part of the Nebula Framework project, released under the MIT License.             *
+* See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
+*************************************************************************************************/
+public class QueryPredicate {
+    private final Schema.SObjectField specificField;
+    private final QueryOperator specificOperator;
+    private final List<Object> specificValues;
+
+    public static QueryPredicate EQUALS(Schema.SObjectField field, Object value) {
```

Comment:
@jamessimone sorry again, but more questions now that I've had a bit more sleep. Is there any difference between..
`QueryPredicate.EQUALS(Lead.Id, expectedLead.Id)`
and this?
`new QueryPredicate.(Lead.Id, QueryOperator.EQUALS, expectedLead.Id)`

It seems like they do the same thing, right?